library/src/main/java/ch/decent/sdk/model/Asset.kt

```diff
@@ -13,22 +13,25 @@ data class Asset(
     @SerializedName("description") val description: String = "",
     @SerializedName("options") val options: Options = Options(),
     @SerializedName("dynamic_asset_data_id") val dataId: ChainObject = ObjectType.NULL_OBJECT.genericId
-): AssetFormatter {
+) : AssetFormatter {
 
   init {
     check(id.objectType == ObjectType.ASSET_OBJECT)
   }
 
-  fun convert(assetAmount: AssetAmount): AssetAmount {
-    if (options.exchangeRate.base.assetId == assetAmount.assetId) {
-      val amount = options.exchangeRate.quote.amount / options.exchangeRate.base.amount * assetAmount.amount
-      return AssetAmount(amount, id)
-    }
-    if (options.exchangeRate.quote.assetId == assetAmount.assetId) {
-      val amount = options.exchangeRate.base.amount / options.exchangeRate.quote.amount * assetAmount.amount
-      return AssetAmount(amount, id)
+  fun convertFromDCT(assetAmount: AssetAmount, roundingMode: RoundingMode) = convert(assetAmount, id, roundingMode)
+
+  fun convertToDCT(assetAmount: AssetAmount, roundingMode: RoundingMode) = convert(assetAmount, DCoreConstants.DCT_ASSET_ID, roundingMode)
```

Comment:
I think we agreed to use `BigInteger` instead of `AssetAmount` to minimise space for confusion (what if wrong asset id is passed to the asset amount parameter?)

Correct me if I'm wrong pls. @petervanderkadecent

Reply:
Make sense. I'll change it.